### DISCUSSION TITLE
How to change dimensions in Clifford algebra using FeynCalc?

### DISCUSSION THREAD

**Discussion by Aletchandre** [FeynCalc/feyncalc#250] [Q&A]
Hello! Does anybody know how to implement FeynCalc for working with Cliffird algebra in 2 dimensions instead of 4? I tried to use the command ChangeDimenssion['obj',2]. But when computing the trace of four Dirac matrices with a gamma5 matrix, the output gives a Levi-Civita tensor with four indices. I'm using FeynCalc 8.2.0 and Mathematica 10.

**Comment by vsht** [comment]
Hi,

this is not supported, so if you need 2-dim you probably would be better 
off with another package.





Am 20.02.24 um 21:49 schrieb Aletchandre:
